compiler/p/codegen/OMRMemoryReference.cpp

```diff
@@ -1522,7 +1522,13 @@ void OMR::Power::MemoryReference::accessStaticItem(TR::Node *node, TR::SymbolRef
          if (!GCRnode)
             GCRnode = cg->getCurrentEvaluationTreeTop()->getNode();
```

Comment:
It doesn't seem like "GCRnode" is the right name for this variable anymore...

Reply:
The comment states `/* don't want to trash node prematurely by the code for handling guarded counting recompilations symbols */` , so I'm not sure what to rename the variable to then :S

Reply:
Sorry, I missed this update before.

So we can't correct the variable name because it's indirectly referenced in a comment :) ? The comment is already wrong! Let's choose not to perpetuate a conservation of confusion law :) .

My point was that `GCRNode` seems to be now used to hold all kinds of different nodes in this function, not *just* nodes that are related to GCR. So why not use a more generic name like `nodeForSymbol` or something like that? The comment can similarly be made more generic "...the code for handling symbol relocations" or some such.